app/views/users/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<% provide :title, "Edit user" %>
+<h1>Update your profile</h1>
```

Comment:
tìm hiểu gem I18n để sửa lại đoạn này, search với keyword " how to use I18n in Rails"